src/foam/u2/borders/CollapseBorder.js

```diff
@@ -95,9 +99,8 @@ foam.CLASS({
             tag(this.TOGGLE, { label: this.label$ }).
           end().
           start('span').
-            // TODO add support for toggle action on title click
             addClass(this.myClass('title')).
-            add(this.title$).
+            tag(this.TOGGLE, { label: this.title$, tabIndex: -1 }).
```

Comment:
Why the tab index? If you just want to focus it then we can do an `.on('load', ()=> { this.el_().focus() })`

Reply:
Also you could just put the button in a `.start()` and then add a togglebutton class to the `start()` so that we can just use `^togglebutton` instead of using the full class name in CSS. Wouldn't really matter but it could affect other buttons called toggle in case they are on the same screen

Reply:
the tabindex was because when I tabbed over the buttons it was kinda weird how the title and the control was looked like two separate buttons, so I want to only be able to tab over the control

Reply:
That's fine then. We can let it be tabIndex -1

Reply:
> Also you could just put the button in a `.start(this.TOGGLE...)` and then add a togglebutton class to the `start()` so that we can just use `^togglebutton` instead of using the full class name in CSS. Wouldn't really matter but it could affect other buttons called toggle in case they are on the same screen

addressed https://github.com/kgrgreer/foam3/pull/587/commits/db3f348d6931ea1101149dc9d04749a5944b2804